pkg/minikube/registry/drvs/hyperv/hyperv.go

```diff
@@ -105,5 +105,20 @@ func status() registry.State {
 		return registry.State{Installed: false, Running: false, Error: errorMessage, Fix: fixMessage, Doc: docURL}
 	}
 
+	// Ensure user is either a Windows Administrator or a Hyper-V Administrator.
+	adminCheckCmd := exec.CommandContext(ctx, path, `@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")`)
+	adminCheckOut, _ := adminCheckCmd.CombinedOutput()
```

Comment:
In Go, all errors must be handled and returned appropriately. Please check the return statement of err.

Reply:
Thank you for the valuable feedback. Addressed with the latest commit.